Graphics_oM/Render/RenderText.cs

```diff
@@ -0,0 +1,65 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+
+using BH.oM.Base;
+using BH.oM.Geometry.CoordinateSystem;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.oM.Graphics
+{
+    [Description("Render text at a location and orientation in space with specified height, font and colour.")]
+    public class RenderText : IRender
+    {
+
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("The text to render.")]
+        public virtual string Text { get; set; } = "";
+
+        [Description("Cartesian to locate and orientate the text.")]
+
+        public virtual Cartesian Cartesian { get; set; } = new Cartesian();
+
+        [Description("Height of the text. Default is 1.")]
```

Comment:
Height in what?

Reply:
I see what you are saying @FraserGreenroyd not completely sure how to address this. Units will be determined by the configuration of the UI rendering the text. So if a user is using Rhino + GH in meters then meters, but if they are using feet then its feet. If this is used to render text in an environment using pixels or points then that is the unit.
Would you be happy with 
```suggestion
        [Description("Height of the text. Default is 1. Units are determined by the settings of the UI rendering the text.")]
```

Reply:
I would be happy with that @rolyhudson 😄